projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/time_based_queue.py

```diff
@@ -36,6 +36,12 @@ def __init__(
         self._dequeue_timeout_seconds = dequeue_timeout_seconds
 
     def enqueue(self, element):
+        """
+        Adds an element to the queue.
+
+        :param element: the element to be added to the queue
```

Comment:
Maybe add some info for what type of element is expected

Reply:
The expected type of element depends on the use case. In the current implementation, we only use it for the DAG's delayed starting jobs but I am not sure if we should add a particular type because we might need the queue for some other objects in the future.

Reply:
Duygu has a point. 

We shoudl use type hints everywhere. For such generic class time TimeBasedQueue (similarly to List, queue, deque, etc) we can use 
A) typing.Any to indicate it can be any
B) Better option -> use generics then user would pick the type `queue = TimeBasedQueue[int]` for example) 

For now A) is good enough.

Reply:
Alright. Done.